draft-ietf-quic-transport.md

```diff
@@ -335,8 +337,9 @@ the flow control limits set by its peer.  Flow control is described in detail in
 Stream multiplexing can have a significant effect on application performance if
 resources allocated to streams are correctly prioritized.
 
-QUIC does not provide frames for exchanging prioritization information.  Instead
-it relies on receiving priority information from the application that uses QUIC.
+QUIC does not provide a mechanism for exchanging prioritization information.
+Instead it relies on receiving priority information from the application that
+uses QUIC.
```

Comment:
```suggestion
uses QUIC in an implementation specific manner.
```

Reply:
implementation-specific, with the hyphen

Reply:
I added this, then removed it because it read somewhat redundantly.